packages/api/src/utils/uploads.ts

```diff
@@ -102,21 +101,18 @@ export const generateUploadFilePathName = (
   return `u/${id}/${fileName}`
 }
 
-export const uploadToSignedUrl = async (
-  uploadUrl: string,
+export const uploadToBucket = async (
+  filePath: string,
   data: Buffer,
-  contentType: string
+  options?: { contentType?: string; public?: boolean },
+  selectedBucket?: string
 ): Promise<void> => {
-  // if (env.dev.isLocal) {
-  //   return
-  // }
+  await storage
```

Comment:
I was thinking about this this morning, and I think we can make these audio files non-public. We just need to returned a signed URL that can download the file when we do the redirect.

Reply:
Yes, it makes sense to me